parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -172,7 +172,7 @@ fi
 
 VALIDATION_ERR=0
 
-API_SERVER_DNS_RETRIES=20
+API_SERVER_DNS_RETRIES=200
 if [[ $API_SERVER_NAME == *.privatelink.* ]]; then
```

Comment:
can probably drop the if here

Reply:
let's keep it here so we know private link has special setting demand. It just happens that we use the same value for public server this time